.github/workflows/release.yml

```diff
@@ -15,14 +15,14 @@ jobs:
       id-token: write
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
       - name: Set up Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v5
         with:
-          go-version: 1.22.2
+          go-version: '1.22'
```

Comment:
Earlier, we were giving the exact patch version, by giving minor version, which patch it is going to install? Will it always pick latest stable patch of minor?

Reply:
It could pick anything, but since we have changed the go line in the go.mod to be `go 1.22.3`, the minimum patch version used will be 1.22.3.